core/services/keystore/keys/tronkey/account.go

```diff
@@ -0,0 +1,175 @@
+package tronkey
+
+import (
+	"crypto/ecdsa"
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/mr-tron/base58"
+)
+
+// Extracted from go-tron sdk: https://github.com/fbsobreira/gotron-sdk
+
+const (
+	// HashLength is the expected length of the hash
+	HashLength = 32
+	// AddressLengthBase58 is the expected length of the address in base58format
+	AddressLengthBase58 = 34
+	// Tron Address Prefix
+	prefixMainnet = 0x41
+	// TronBytePrefix is the hex prefix to address
+	TronBytePrefix = byte(prefixMainnet)
+	// Tron address should should have 20 bytes + 4 checksum + 1 Prefix
+	AddressLength = 20
+)
+
+// Address represents the 21 byte address of an Tron account.
+type Address []byte
```

Comment:
If it is fixed at 21 bytes, did you consider using an array for the underlying type?
```suggestion
// Address represents the 21 byte address of an Tron account.
type Address [21]byte
```